hooks/post/30-upgrade.rb

```diff
@@ -1,4 +1,6 @@
 if app_value(:upgrade)
+  noop = app_value(:noop) ? ' (noop)' : ''
```

Comment:
Ahh, yea thanks for pointing that out. I figured there was something I just wasn't seeing it.